packages/core/src/odata-v2/entity.spec.ts

```diff
@@ -1,114 +0,0 @@
-import { TestEntity } from '../../test/test-util/test-services/v2/test-service';
```

Comment:
I assume you move the tests from here to the common.

Reply:
Yes.